doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,40 @@
+# Ansible/Helm add api command  Proposal for Operator SDK
```

Comment:
@bharathi-tenneti the template that we need to use is: https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/TEMPLATE.md

Reply:
I was looking at previous proposal docs, https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/ansible-operator.md